### ISSUE TITLE
Roadmap for openCypher support?

### ISSUE THREAD

**Issue by edsar** [graphframes/graphframes#156]
Exciting news here for folks who have worked with neo4j over the years. 

http://www.opencypher.org/

Is Databricks or this team (or someone else) going to drive the support for openCypher in Spark?

Awesome work so far!

**Comment by thunterdb** [comment]
Yes, this is a welcome development. We currently do not have the resources to work on this at Databricks, but supporting a subset of the most popular operations would be a great start.
